bot/migrations/2020-02-15-Initial.js

```diff
@@ -0,0 +1,37 @@
+module.exports.up = function() {
```

Comment:
This is what SQLite Studio gave me when I exported the template database.
Is there a way to diff database schemas, so we can see if nothing was messed up?
Did the template database include any data by default which we need to add here (BotSettings come to mind, but I haven't checked - shouldn't the code itself handle such cases anyway)?